deployment-assets/natural-language-understanding-code-pattern/helm/naturallanguageunderstandingmonitoredhelmpr/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: naturallanguageunderstandingmonitoredhelmpr
```

Comment:
Do these long names cause issues at deploy time? I seem to recall Helm has limits.

Reply:
Shouldnt, length requirement is 63 char I think